docs/change-log/index.md

```diff
@@ -1,14 +1,95 @@
+| --- | ----------------------------------------------------------------- |
+| ML-2296 | Add ability to manage datastore credentials with datastore profiles. Supports Redis. See [Using data store profiles](../store/datastore.html#using-data-store-profiles), [view in Git](https://github.com/mlrun/mlrun/pull/3936). |
+| ML-3500 | Supports a dbfs data store (Databricks file system) data store. See [Databricks file system](../store/datastore.html#databricks-file-system), [view in Git](https://github.com/mlrun/mlrun/pull/3626).|
+
+### Feature store
+| ID    | Description                                                    |
+| --- | ----------------------------------------------------------------- |
+| ML-3784 | Supports feature vector-defined feature-set relations and join-type (per-join). Tech-preview. See [Feature vector with different entities and complex joins](../feature-store/feature-vectors.html#feature-vector-with-different-entities-and-complex-joins) and [view in Git](https://github.com/mlrun/mlrun/pull/3947).|
+
+### Infrastructure
+| ID    | Description                                                    |
+| --- | ----------------------------------------------------------------- |
+| ML-3370 | Accessing the MLRun hub is now through a service API. This will enable implementing better function version selection and combining hub functions from different sources. Tech-preview. [View in Git](https://github.com/mlrun/mlrun/pull/3384).|
+| ML-3644 | Supports self-signed docker registries. See [Using self-signed registry](../runtimes/image-build.html#using-self-signed-registry) and [view in Git](https://github.com/mlrun/mlrun/pull/4013). |
+| ML-4132 | New remote function `http_client_kwargs` used to pass any parameter supported in the requests.request method. See [invoke](../api/mlrun.runtimes.html#mlrun.runtimes.RemoteRuntime.invoke) and [view in Git](https://github.com/mlrun/mlrun/pull/3872). |
+
+
+### Runtimes
+| ID    | Description                                                    |
+| --- | ----------------------------------------------------------------- |
+| ML-3501 | Supports running Spark jobs on Databricks cluster. See {ref}`databricks`. [View in Git](https://github.com/mlrun/mlrun/pull/3996). |
+| ML-3854 | Add webhook notification. See webhook in [Notification Kinds](../concepts/notifications.html#notification-kinds) and [view in Git](https://github.com/mlrun/mlrun/pull/3946). |
+| ML-4059 | Supports adding env vars or secrets to the docker build during runtime. See [Extra arguments](../runtimes/image-build.html#extra-arguments),  {py:class}`~mlrun.projects.html#mlrun.projects.MlrunProject.build_config` and [view in Git](https://github.com/mlrun/mlrun/pull/4013). |
+
+
+### UI
+| ID    | Description                                                    |
+| --- | ----------------------------------------------------------------- |
+| ML-2811 | New Batch Inference wizard. Tech Preview.|
```

Comment:
maybe move the tech previews to the bottom

Reply:
They are in numerical order